code/__DEFINES/preferences.dm

```diff
@@ -26,5 +26,6 @@
 #define BE_PAI			128
 #define BE_CULTIST		256
 #define BE_BLOB			512
-#define BE_NINJA		1024
-#define BE_ANGEL		2048
\ No newline at end of file
+#define BE_POSIBRAIN	1024
+#define BE_ANGEL		2048
+#define BE_NINJA		4096
```

Comment:
I haven't messed with bitflags enough but i think there might be a slight issue if you are redefining ninja and angel instead of just slaping the brain in 4096